Sources/Plasma/FeatureLib/pfMoviePlayer/plMoviePlayer.cpp

```diff
@@ -460,6 +469,9 @@ bool plMoviePlayer::Stop()
         fAudioSound->Stop();
     if (fPlate)
         fPlate->SetVisible(false);
+    if (fLastImg) {
+        vpx_img_free(fLastImg);
```

Comment:
Do we need to assign `nullptr` to `fLastImg` here to prevent it from potentially getting double freed during the frame loop?

Reply:
I think so. Good catch! Wondering is this is the source of the crash on Windows but the project is still configuring.

Reply:
Actually this probably isn't the Windows crash. That happens in normal frame playback.